src/applications/debt-letters/components/DebtLettersWrapper.jsx

```diff
@@ -4,28 +4,60 @@ import LoadingIndicator from '@department-of-veterans-affairs/formation-react/Lo
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import CallToActionWidget from 'platform/site-wide/cta-widget';
 import { bindActionCreators } from 'redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import { fetchDebtLetters, fetchDebtLettersVBMS } from '../actions';
 
 class DebtLettersWrapper extends Component {
   componentDidMount() {
-    this.props.fetchDebtLetters();
-    this.props.fetchDebtLettersVBMS();
+    if (this.props.showDebtLetters !== false) {
+      this.props.fetchDebtLetters();
+      this.props.fetchDebtLettersVBMS();
+    }
   }
 
-  renderError = () => (
-    <AlertBox headline="Error alert" content="Temp error text" status="error" />
-  );
-
   render() {
-    const { isPending, isPendingVBMS, children, isError } = this.props;
+    const {
+      isPending,
+      isPendingVBMS,
+      children,
+      isError,
+      isErrorVBMS,
+      showDebtLetters,
+    } = this.props;
+
+    if (showDebtLetters === false) {
+      return document.location.replace('/');
+    }
```

Comment:
The condition looks weird but this fixes an edge case where a user visits the URL directly - and on the initial component load, the featureFlags are not set so it returns undefined / falsey - however, when we compare it to false only, undefined will not trigger the redirect.

Reply:
I think using `window.location` is the preferred method here.

[`document`](https://developer.mozilla.org/en-US/docs/Web/API/Document/location) vs [`window`](https://developer.mozilla.org/en-US/docs/Web/API/Window/location)